drivers/CoreVideo/include/interface/Font.hpp

```diff
@@ -4,27 +4,25 @@
 
 #pragma once
 
-// #include <cstdint>
+#include <cstdint>
 
-#include "CGColor.hpp"
-#include "CGPixel.hpp"
+namespace leka {
+
+struct CGColor;
+struct Character;
+
+}	// namespace leka
 
 namespace leka::interface {
 
 class Font
 {
   public:
-	struct Character {
-		CGPoint origin {};	 // Top left corner by convention
-		uint8_t ascii {};	 // From 0x20 to 0x7F
-	};
-
 	virtual ~Font() = default;
 
-	virtual void drawChar(Character character, CGColor foreground = CGColor::black,
-						  CGColor background = CGColor::white) = 0;
-	virtual void display(const char *text, uint32_t size, uint32_t starting_line, CGColor foreground = CGColor::black,
-						 CGColor background = CGColor::white)  = 0;
+	virtual void drawChar(Character character, CGColor foreground, CGColor background) = 0;
```

Comment:
the same question about background/foreground can be asked here: could they be stored directly inside the struct?